webextensions/api/menus.json

```diff
@@ -438,6 +438,37 @@
               }
             }
           },
+          "srcUrl": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": true
+                },
+                "edge": {
+                  "version_added": "14"
+                },
+                "firefox": [
+                  {
+                    "version_added": "48"
+                  },
+                  {
+                    "version_added": "59",
+                    "version_removed": "94",
+                    "notes": "Returns a post-redirect URL rather than the raw <code>&lt;img src&gt;<code> value."
```

Comment:
Not limited to `<img src>`, please make this more generic. See https://github.com/mdn/content/pull/10214#pullrequestreview-795673940